haystack/schema.py

```diff
@@ -94,17 +96,52 @@ def __str__(self):
         return str(self.to_dict())
 
 
+@dataclass
+class Answer:
+    answer: Optional[str]
```

Comment:
How we can this while keeping multimod in the mind?

Reply:
Not sure if I understood your question. Do you mean if the answer is not a string? What do you have in mind here? IMO for QA this limitation is okay until we have a clearer plan how to integrate visualQA that is pointing to "images" and a like.  For `Documents` this is different and much clearer. We might have an image search that returns "image documents" as results or takes them as an query input. We should cope for that already now...

Reply:
Okay make sense